src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -243,13 +244,11 @@ void CheckELUDerivativeCorrect(const arma::colvec input,
   ELU<> lrf(1.0);
 
   // Test the calculation of the derivatives using the entire vector as input.
-  arma::colvec derivatives, activations;
+  arma::colvec derivatives;
```

Comment:
Not sure I see the reason for the changes here, can you elaborate on that one?

Reply:
I am sorry, that was  changed unknowingly.